### ISSUE TITLE
Update developer documentation to mention contributing web-components - Documentation

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#1007]
## Description
Katy and I reviewed our Contribution guidelines and Katy pointed out that the documentation for Developers does not direct them to build new components as web-components. We should update our guidance to instruct developers to contribute web-components when building a new component. 

## Details
Update this page: https://design.va.gov/about/developers#contributing-experimental-design-code

## Tasks
- [ ] Write or update component documentation for design.va.gov
- [ ] Coordinate with accessibility specialist to create or update accessibility guidance
- [ ] Request documentation draft review
- [ ] Once documentation is approved, publish to design.va.gov

## Acceptance Criteria
- [ ] Updated documentation is published on design.va.gov

**Comment by humancompanion-usds** [comment]
We no longer have any other kind of component besides web components so this is no longer relevant. I've reviewed the documentation and this is accurate: https://design.va.gov/about/developers/contributing
